documentation/hpcc-tf-for-developers.md

```diff
@@ -0,0 +1,373 @@
+# For Developers: Tutorial of HPCC Easy Deploy Terraform 
+
+This tutorial explains the terraform that deploys HPCC Systems on an azure kubernetes service (aks). The terraform was designed to enable one to deploy HPCC Systems easily.
+The terraform can be found on github. Here is a link to it ([https://github.com/hpccsystems-solutions-lab/terraform-azurerm-hpcc-lite])
+
+From the root directory of the repository one can deploy all components of the HPCC cluster. Also, one can deploy individual components of the system from these subdirectories: `vnet`, `storage`, `aks`, and `hpcc`. If you want to deploy the individual components manually, here is the order you should do the deployment: 1st `vnet`, 2nd `storage` (if you want persistent storage), 3rd `aks`, and finally `hpcc`.
+
+The following sections will explain the terraform in root directory and all subdirectories.
```

Comment:
Minor: The phrase 'the terraform' confused me until I realized that you really mean 'the Terraform code'.  Maybe change it throughout the document for clarity?

Reply:
Fixed